src/player.cpp

```diff
@@ -443,6 +443,30 @@ void Player::setVolume(SoLoud::handle handle, float volume)
     return soloud.setVolume(handle, volume);
 }
 
+float Player::getPan(SoLoud::handle handle)
+{
+    return soloud.getPan(handle);
+}
+
+void Player::setPan(SoLoud::handle handle, float pan)
+{
+    if (pan > 1.0f)
+        pan = 1.0f;
+    if (pan < -1.0f)
+        pan = -1.0f;
```

Comment:
Oh, I see that we're sanitizing the input here in C code. Since this is likely not a performance bottleneck, I'd rather sanitize it in Dart land, since that's closer to the user's debugger.

Reply:
You are right! Better to let the dev know what he is trying to do :)